src/forms/EditDocumentForm.js

```diff
@@ -17,98 +16,107 @@ const EditDocumentForm = ({
   onNameChange,
   onDescriptionChange,
   onVersionStatusChange,
+  handleSubmit,
+  errors,
+  submitButtons,
 }) => {
-  const versionStatusList = Object.keys(versionState);
-  const dropdownClass = classes('Dropdown');
-  return (
-    <form className="pt-16">
-      <GridContainer>
-        <label
-          className="Form--Label row-1 cell-3 text-right Form--Label-required"
-          htmlFor="file_name"
+  const [newFileType, setFileType] = useState();
+  const [newFileName, setFileName] = useState();
+  const [newFileDescription, setFileDescription] = useState();
+  const [onUploading, setUploading] = useState(false);
+  const options = [];
+  Object.keys(versionState).forEach(function(e) {
+    options.push({
+      key: e,
+      value: e,
+      text: versionState[e].title,
+      content: (
```

Comment:
I _think_ that `content: <Badge state={state} />` is valid here without the extra `<button />`